README.md

```diff
@@ -321,6 +322,11 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 * [hEngine](https://github.com/hashintel/hash/tree/main/packages/engine) - A Rust-implemented computational simulation engine, supporting large-scale agent-based modelling, with simulation logic written in JavaScript and Python.
 
+### Social networks
+
+* Mastodon
+  * [Rustodon](https://github.com/rustodon/rustodon) - A Mastodon-compatible, ActivityPub-speaking server in Rust [![Build Status](https://travis-ci.org/rustodon/rustodon.svg?branch=master)](https://travis-ci.org/rustodon/rustodon)
```

Comment:
The Travis image link 301's to https://api.travis-ci.org/rustodon/rustodon.svg?branch=master which is breaking the CI (because we assume that 301's will go away at some point and so need fixing to be the actual end link)